examples/with-okta-multi-tenant-pkce-flow/api-server/utils.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"sync"
+
+	verifier "github.com/nirasan/go-oauth-pkce-code-verifier"
+)
+
+var oktaVerifiers = sync.Map{}
+
+func getOrCreateVerifierForState(state string) *verifier.CodeVerifier {
+	// Since we are using the PKCE code flow, we need to persist the verifier for the state
```

Comment:
Add a TODO in the comment explaining the map needs to be made in the db. 1:1 in the db, and revoke once it's used.

Also make another function to revoke the verifier based on the state input -> this should be called in the API override for signinup

Reply:
done